CMakeLists.txt

```diff
@@ -693,6 +693,8 @@ if(NOT NO_CODEGEN)
         "${BUN_SRC}/js/thirdparty/*.ts"
         "${BUN_SRC}/js/internal/*.js"
         "${BUN_SRC}/js/internal/*.ts"
+        "${BUN_SRC}/js/internal/*/*.js"
+        "${BUN_SRC}/js/internal/*/*.ts"
```

Comment:
Fancy glob support here would be nice (I know it's limited in CMake but this has me dreaming).